panther_bringup/config/panther_default.yaml

```diff
@@ -0,0 +1,29 @@
+panther:
+  config_version: "0.1"
+  serial_no: ""
+  robot_version: "1.0"
+
+  imu:
+    model: "PhidgetSpatial_1042_0B"
+    localization:
+      x: 0.195
+      y: -0.01
+      z: 0.085
+    orientation:
+      r: 3.14159
+      p: 0.0
+      y: 3.14159
```

Comment:
This requires double check

Reply:
This config is for Panther 1.0.5
Parameters for 1.0 can be found here:
https://github.com/husarion/panther_driver/blob/1040796540c970d60c88310f96c7181c263ba747/launch/phidgets_imu.launch#L20

Reply:
The values you send seems to be good.